journalpump/senders/elasticsearch.py

```diff
@@ -10,7 +10,8 @@
 class ElasticsearchSender(LogSender):
     def __init__(self, *, config, **kwargs):
         super().__init__(config=config, max_send_interval=config.get("max_send_interval", 10.0), **kwargs)
-        self.session_url = self.config.get("elasticsearch_url")
+        # ES and OS are sensitive to multiple slashes in the path
+        self.session_url = self.config.get("elasticsearch_url", "").rstrip("/")
```

Comment:
Now this will contain a `""` value if `config` doesn't contain an `"elasticsearch_url"` key - perhaps better to do it conditionally on `is not None`. Conversely if the key is always present, replacing `config.get(key)` with `config[key]`

Reply:
It _should_ be present, but as there is currently no wider config validation, it _could_ be empty. But, maybe better to throw a `KeyError` in that case ¯\_(ツ)_/¯.